sdk/internal/CHANGELOG.md

```diff
@@ -1,14 +1,10 @@
 # Release History
 
-## 1.0.2 (Unreleased)
+## 1.1.0 (2022-10-19)
```

Comment:
This will need to change to today's date.

Reply:
@jhendrixMSFT thanks, done